### PR TITLE
chore: Manifest setup with single root component

### PR DESCRIPTION
**Pull request by ddixit14**

### CODE REVIEW

**Review by suztomo — APPROVED** [review]
Adding stamp but wait for Jeff

File: release-please-config.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  "release-type": "java-yoshi",
+  "versioning": "always-bump-minor",
+  "separate-pull-requests": false,
+  "include-component-in-tag": false,
+  "packages": {
+    ".": {
+      "component": "main"
+    }
+  },
+  "plugins": ["maven-workspace"]
```
Comment (chingor13) [inline release-please-config.json:11]: You should skip this plugin if there's only a single component unless you have pom.xml files without annotations.

It shouldn't hurt but will make extra API calls and do extra work to build the dependency tree.

**Review by chingor13 — APPROVED** [review]

File: release-please-config.json (hunk 1 above)
Comment (ddixit14) [reply release-please-config.json:11]: @meltsufin, I remember you fixed all the pom.xml's to have the annotations. Should I remove this plugin then? or better to have it around for now?

Comment (meltsufin) [reply release-please-config.json:11]: All poms are fixed to have annotations and there is a check to verify it.

**Pull request merged**